justaddcl/yujinelson.com
# [CU-2x1wjgn] Update home page for 2022

## Background
A lot of the copy on the home page is outdate and doesn't reflect the role at Duffel or the move to the UK. I need this updated to accurately reflect my location and work experience.

## Updates
- Added Duffel role to experience data
- Updated Magpul experience data to include an end date
- Added an `ExperienceCard` to display my role at Duffel
- Added Duffel logos to use for displaying work experience
- Added 'duffel' to `spellcheck` list
- Updated copy to reflect changes on the updated CV

## How to test
- Make sure the copy updates are reflected
- Ensure there is a new `ExperienceCard` for my Duffel role
